databags/databag_timer.h

```diff
@@ -0,0 +1,53 @@
+﻿#pragma once
+
+#include "core/math/math_defs.h"
+#include "core/templates/local_vector.h"
+#include "databag.h"
+
+typedef uint64_t godex::Timer; // μs (microseconds) until the timer fires
+typedef uint32_t godex::TimerHandle;
+
+class TimersDatabag : public godex::Databag {
+	DATABAG(TimersDatabag)
+
+	static void _bind_methods();
+
+	LocalVector<godex::Timer, godex::TimerHandle> timers;
+	LocalVector<godex::TimerHandle, godex::TimerHandle> destroyed_timers;
+	uint64_t now; // In μs (microseconds)
+
+	void set_now(const uint64_t newNow);
+
+	//
```

Comment:
In godex I'm using the triple slash when I want to add a documenting comment `///`. It would be nice if you can use it here too, so we keep it coherent with the rest.